packages/react/src/ReactContext.js

```diff
@@ -20,16 +20,15 @@ export function createContext<T>(
 ): ReactContext<T> {
   if (calculateChangedBits === undefined) {
     calculateChangedBits = null;
-  } else {
-    if (__DEV__) {
-      warningWithoutStack(
-        calculateChangedBits === null ||
-          typeof calculateChangedBits === 'function',
-        'createContext: Expected the optional second argument to be a ' +
-          'function. Instead received: %s',
-        calculateChangedBits,
-      );
-    }
+  }
+  if (__DEV__) {
+    warningWithoutStack(
+      calculateChangedBits === null ||
```

Comment:
this change could cause this warning to be thrown now where it wouldn't before, `calculateChangedBits` is `undefined`, then gets set to `null`, then you are in `__DEV__` and `calculateChangedBits === null` now passes.

Reply:
`warningWithoutStack` fires when its first argument is falsey. When `calculateChangedBits` is `null`, it does not warn.

So in either version, if `calculateChangedBits` is undefined, there's no warning. So the behavior is the same.

Reply:
ah I assumed it called when `true`, my bad.